### PR TITLE
[Service Connector]: update to API version 2022-11-01-preview

### PR DESCRIPTION
**Pull request by xfz11**
## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1 [hunk 1]
```diff
@@ -56,27 +56,101 @@ function New-AzServiceLinkerForContainerApp {
...
+        [Parameter()]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Category('Body')]
+        [Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Runtime.Info(PossibleTypes = ([Microsoft.Azure.PowerShell.Cmdlets.ServiceLinker.Models.Api20221101Preview.IConfigurationInfoCustomizedKeys]))]
+        [System.Collections.Hashtable]
+        # Optional.
+        # A dictionary of default key name and customized key name mapping.
+        # If not specified, default key name will be used for generate configurations
+        ${ConfigurationInfoCustomizedKey},
+    
+        [Parameter(DontShow)]
```
Comment (BethanyZhou) [inline src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1:106]: why you mark parameter as DontShow

File: src/ServiceLinker/docs/Get-AzServiceLinkerConfigurationName.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+Get-AzServiceLinkerConfigurationName [-Filter <String>] [-SkipToken <String>] [-DefaultProfile <PSObject>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Lists the configuration names generated by Service Connector for all target, client types, auth types.
+
+## EXAMPLES
+
+### Example 1: {{ Add title here }}
```
Comment (BethanyZhou) [inline src/ServiceLinker/docs/Get-AzServiceLinkerConfigurationName.md:25]: please add example for each cmdlet

**Review by BethanyZhou — CHANGES_REQUESTED** [review]

File: src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1 (hunk 1 above)
Comment (xfz11) [reply src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1:106]: This feature is required by some specific scenario and we decide to not to expose it on powershell

File: src/ServiceLinker/docs/Get-AzServiceLinkerConfigurationName.md (hunk 2 above)
Comment (xfz11) [reply src/ServiceLinker/docs/Get-AzServiceLinkerConfigurationName.md:25]: Updated

File: src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1 (hunk 1 above)
Comment (BethanyZhou) [reply src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1:106]: you can just comment this parameter from customized cmdlet

Comment (xfz11) [reply src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1]: Ok. The internal cmdlet has no param like this with customization directive in readme file. I remove the param as well in custom files.

Comment (BethanyZhou) [reply src/ServiceLinker/custom/New-AzServiceLinkerForContainerApp.ps1]: Please let me know if your PR is ready to review again by labelling `need-review`

**Review by BethanyZhou — APPROVED** [review]

**Pull request merged**